docs/qsimcirq_tutorial.ipynb

```diff
@@ -0,0 +1,636 @@
+        "cirq_results = cirq_simulator.simulate(circuit)\n",
+        "print(cirq_results)\n",
+        "print()\n",
+        "\n",
+        "# Simulate the circuit with qsim and return the full state vector.\n",
+        "print('qsim results:')\n",
+        "qsim_simulator = qsimcirq.QSimSimulator()\n",
+        "qsim_results = qsim_simulator.simulate(circuit)\n",
+        "print(qsim_results)"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Cirq results:\n",
+            "measurements: (no measurements)\n",
+            "output vector: 0.707|00⟩ + 0.707|11⟩\n",
+            "\n",
+            "qsim results:\n",
+            "measurements: (no measurements)\n",
+            "output vector: 0.707|00⟩ + 0.707|11⟩\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-kyXlKujtZcp",
+        "colab_type": "text"
+      },
+      "source": [
+        "To sample from this state, you can invoke Cirq's `sample_state_vector` method:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "-m6uzQ6ms9I7",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 192
+        },
+        "outputId": "0c5a4363-bb23-4602-8c50-1f8935d9ac0c"
+      },
+      "source": [
+        "samples = cirq.sample_state_vector(\n",
+        "    qsim_results.state_vector(), indices=[0, 1], repetitions=10)\n",
+        "print(samples)"
+      ],
+      "execution_count": 3,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "[[1 1]\n",
+            " [0 0]\n",
+            " [1 1]\n",
+            " [1 1]\n",
+            " [0 0]\n",
+            " [0 0]\n",
+            " [0 0]\n",
+            " [1 1]\n",
+            " [1 1]\n",
+            " [1 1]]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "pOhybi46sHwI",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Measurement sampling\n",
+        "\n",
+        "qsim also supports sampling from user-defined measurement gates. Note that since qsim and Cirq use different random number generators, identical runs on both simulators may give different results, even if they use the same seed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "NRJvtqYrnylJ",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 157
+        },
+        "outputId": "0c5ec393-116e-4c43-c090-3d680ca159a6"
+      },
+      "source": [
+        "# Define a circuit with measurements.\n",
+        "q0, q1 = cirq.LineQubit.range(2)\n",
+        "circuit = cirq.Circuit(\n",
```

Comment:
I think it would be useful to print out the circuit diagram for people to look at